README.RDAP.md

```diff
@@ -45,21 +45,22 @@ For example: https://rdap.db.ripe.net/entity/KR4422-RIPE
 Related Contact information is Filtered
 ---------------------------------------
 All contact entities ("technical","administrative" etc.) have filtered contact information, i.e. "e-mail"
```

Comment:
Add the word "related" to the first sentence, i.e. "All related entities... have filtered contact information..."